<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1316</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QOpenGLWidget" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>960</width>
      <height>720</height>
     </rect>
    </property>
   </widget>
   <widget class="QOpenGLWidget" name="openGLWidget_2">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>10</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_device">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>300</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_openCam">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>300</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>打开相机:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_changeCam">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>350</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>调节相机:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1316</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
